docs/files.md

```diff
@@ -746,3 +747,26 @@ A file's watermark can be removed by calling
 ```js
 client.files.removeWatermark('67890', callback);
 ```
+
+Get Representation
+------------------
+
+A file's representation can be retrieved by calling
+[`files.getRepresentations(fileID, [representationTypes], options,
+callback)`](https://opensource.box.com/box-node-sdk/Files.html#getRepresentation).
+You will be able to fetch a pdf representation, thumbnail representation, multi-page images
+representation, and fetching text representation.
+
+You can fetch different dimensions for the representation by calling
+```js
+client.files.getRepresentationInfo('67890', client.files.representation.PDF, null, callback);
```

Comment:
This example doesn't seem to fit with the sentence above; what dimensions can they fetch here?